src/components/Dropdown/_Dropdown.scss

```diff
@@ -0,0 +1,80 @@
+.dropdown {
+  color: inherit;
+  display: inline-block;
+
+  &__content {
+    background-color: var(--ui-white);
```

Comment:
This should be inherited naturally, no? Can we remove this?

Reply:
It's see through for some reason. But the line you commented on below is redundant. Will remove that.

Reply:
When we have the content underneath we should be able to test this more carefully—it's only a line so it's not hurting anybody, it's just odd that we're seeing different things